DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: miic
 Title: Learning Causal or Non-Causal Graphical Models Using Information Theory
-Version: 1.4.2.9000
+Version: 1.5.0
```

Comment:
- Next time try to push the version number in a separate commit right before the submit.